### DISCUSSION TITLE
Searching for multiple words has no results

### DISCUSSION THREAD

**Discussion by JetGum** [datenstrom/community#981] [Report a bug] [answered]
How to reproduce:

1. fresh install of Yellow Datenstrom
Datenstrom Yellow 0.9
Core 0.9.10
Edit 0.9.9
English 0.9.1
Generate 0.9.6
German 0.9.1
Image 0.9.3
Markdown 0.9.1
Serve 0.9.1
Stockholm 0.9.2
Swedish 0.9.1
Update 0.9.4

2. Install Search (Search 0.9.2).
3. On localhost, go to /search/ and search for one word:
![image](https://github.com/user-attachments/assets/d7ef151b-45ee-41d9-9b39-984cea1edd2b)

The result URL looks like /search/query:lorem/.

4. But search for multiple words fails (with or without quotation marks):
![image](https://github.com/user-attachments/assets/d029e83f-569b-425d-a77b-bbd91a7dccd4)

The result URL looks like /search/query:lorem-ipsum/.

One hint: If I change the URL into /search/query:lorem%20ipsum/, the results display fine.

**Comment by JetGum** [comment]
Some digging into core.php, et voila:

// Normalise location arguments
public function normaliseArguments($text, $filterStrict = true) {
    if ($filterStrict) {
        **// $text = str_replace(" ", "-", strtoloweru($text)); // Remove this line if you don't want spaces converted to dashes**
        **$text = strtoloweru($text); // Convert to lowercase if needed**
    }

**Comment by markseuffert** [comment, marked as answer]
Thank you for the detailed bug report. The bug was introduced three months ago. I wanted to simplify the Yellow API, wanted to make the handling of location arguments easier, but forgot to pass an argument. Sorry about that. Should be fixed in latest [core](https://github.com/annaesvensson/yellow-core), please try it out.

**Comment by JetGum** [comment]
Thanks, @markseuffert! Can confirm, the latest core 0.9.11 works perfectly.
